evm-contracts/src/v0.6/dev/ConversionProxy.sol

```diff
@@ -0,0 +1,152 @@
+    to = AggregatorInterface(_to);
+    emit AddressesUpdated(
+      _from,
+      _to
+    );
+  }
+
+  /**
+   * @notice Converts the latest answer of the `from` aggregator
+   * to the rate of the `to` aggregator
+   * @return The converted answer with amount of precision as defined
+   * by `decimals` of the `to` aggregator
+   */
+  function latestAnswer() external view virtual override returns (int256) {
+    return _latestAnswer();
+  }
+
+  /**
+   * @notice Calls the `latestTimestamp()` function of the `from`
+   * aggregator
+   * @return The value of latestTimestamp for the `from` aggregator
+   */
+  function latestTimestamp() external view virtual override returns (uint256) {
+    return _latestTimestamp();
+  }
+
+  /**
+   * @notice Calls the `latestRound()` function of the `from`
+   * aggregator
+   * @return The value of latestRound for the `from` aggregator
+   */
+  function latestRound() external view virtual override returns (uint256) {
+    return _latestRound();
+  }
+
+  /**
+   * @notice Converts the specified answer for `_roundId` of the
+   * `from` aggregator to the latestAnswer of the `to` aggregator
+   * @return The converted answer for `_roundId` of the `from`
+   * aggregator with the amount of precision as defined by `decimals`
+   * of the `to` aggregator
+   */
+  function getAnswer(uint256 _roundId) external view virtual override returns (int256) {
+    return _getAnswer(_roundId);
+  }
+
+  /**
+   * @notice Calls the `getTimestamp(_roundId)` function of the `from`
+   * aggregator for the specified `_roundId`
+   * @return The timestamp of the `from` aggregator for the specified
+   * `_roundId`
+   */
+  function getTimestamp(uint256 _roundId) external view virtual override returns (uint256) {
+    return _getTimestamp(_roundId);
+  }
+
+  /**
+   * @notice Calls the `decimals()` function of the `to` aggregator
+   * @return The amount of precision the converted answer will contain
+   */
+  function decimals() external view override returns (uint8) {
+    return to.decimals();
+  }
+
+  function _latestAnswer() internal view returns (int256) {
+    return convertAnswer(from.latestAnswer(), to.latestAnswer());
+  }
+
+  function _latestTimestamp() internal view returns (uint256) {
+    return from.latestTimestamp();
+  }
+
+  function _latestRound() internal view returns (uint256) {
+    return from.latestRound();
+  }
+
+  function _getAnswer(uint256 _roundId) internal view returns (int256) {
+    return convertAnswer(from.getAnswer(_roundId), to.latestAnswer());
+  }
+
+  function _getTimestamp(uint256 _roundId) internal view returns (uint256) {
+    return from.getTimestamp(_roundId);
+  }
+
+  /**
+   * @notice Converts the answer of the `from` aggregator to the rate
+   * of the `to` aggregator at the precision of `decimals` of the `to`
+   * aggregator
+   * @return The converted answer
+   */
+  function convertAnswer(
+    int256 _answerFrom,
+    int256 _answerTo
```

Comment:
There is already an implicit assumption that we're converting to `to`'s value, because we're converting to `to`'s decimal precision. I'd prefer to cut the `_answerTo` parameter and just make the `to.latestAnswer()` call directly in this helper.